products/mxlinux.md

```diff
@@ -22,68 +22,78 @@ auto:
 -   distrowatch: mx
     regex: '^Distribution Release: MX Linux (?P<major>\d+)(\.(?P<minor>\d+))?$'
 
+# EOL dates documented on https://mxlinux.org/previous-releases/
 releases:
+-   releaseCycle: "23"
+    codename: "Libretto"
+    releaseDate: 2023-07-31
+    support: true
```

Comment:
Should we use the same dates as Debian 12 here?

That's what the page has currently been following.

Reply:
Done.